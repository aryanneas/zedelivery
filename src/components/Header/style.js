import styled from 'styled-components'

export const HeaderContent = styled.div`
  background-color: rgb(51, 51, 51);
  text-align: center;
  padding: 10px 0;
`

export const Logo = styled.img`
  width: 10%;
  height: 10%;
  @media (max-width: 600px) {
    width: 45%;
  }
`
