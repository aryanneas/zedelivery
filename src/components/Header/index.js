import React from 'react'
import { HeaderContent, Logo } from './style'

const Header = () => {
  return (
    <HeaderContent>
      <Logo
        src="https://courier-images-web.imgix.net/static/img/white-logo.png?auto=compress,format&fit=max&w=undefined&h=undefined&dpr=2&fm=png"
        title="logo ZÃ© Delivery"
        alt="logo ZÃ© Delivery"
      />
    </HeaderContent>
  )
}

export default Header
