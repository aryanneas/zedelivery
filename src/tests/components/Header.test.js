import React from 'react'
import { render } from '@testing-library/react'

import Header from '../../components/Header'

describe('Header component', () => {
  it('should render Header component correctly', async () => {
    const { asFragment } = render(<Header />)
    expect(asFragment()).toMatchSnapshot()
  })
})
