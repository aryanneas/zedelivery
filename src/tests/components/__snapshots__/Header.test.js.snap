// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component correctly 1`] = `
<DocumentFragment>
  .c0 {
  background-color: rgb(51,51,51);
  text-align: center;
  padding: 10px 0;
}

.c1 {
  width: 10%;
  height: 10%;
}

@media (max-width:600px) {
  .c1 {
    width: 45%;
  }
}

<div
    class="c0"
  >
    <img
      alt="logo Zé Delivery"
      class="c1"
      src="https://courier-images-web.imgix.net/static/img/white-logo.png?auto=compress,format&fit=max&w=undefined&h=undefined&dpr=2&fm=png"
      title="logo Zé Delivery"
    />
  </div>
</DocumentFragment>
`;
